packages/sourcemap-upload/src/index.ts

```diff
@@ -10,13 +10,14 @@ import crypto from "crypto";
 import assert from "assert";
 
 import fetch, { Response } from "node-fetch";
+import pMap from "p-map";
 import makeDebug from "debug";
 import glob from "glob";
 import matchAll from "string.prototype.matchall";
 
 const globPromisified = util.promisify(glob);
 
-const debug = makeDebug("recordreplay:sourcemap-upload");
+const debug = makeDebug("replay:sourcemap-upload");
```

Comment:
Probably should change the reference in the [readme](https://github.com/replayio/replay-cli/blob/6a39b90eeefa610360dc3711f242e8276497f0cf/packages/sourcemap-upload/README.md?plain=1#L15)